![alt text](https://github.com/solouli/Pangenomics_Adaptations/blob/master/header.png)

# Characterization of local adaptations in the genus Virgibacillus through pangenomic analysis
## Navarro-Miranda M. (IE-UNAM), García-Ulloa M., Viladomat-Jasso M., Souza V.

**Introduction**

Evolution, metabolism and molecular mechanisms underlying extreme adaptations has changed the way we understand microbial interactions, community roles and how the environment leads to functional diversification even in closely phylogenetically related microorganisms (Shapiro & Polz, 2014). In this work, we performed a pangenomic and functional analysis of the genus *Virgibacillus* using 34 genomes available in public databases and a new *Virgibacillus* genome recently isolated from a high salinity site in Cuatro Ciénegas, Coahuila. Extremophilic microorganisms like *Virgibacillus* are excellent models to look for molecular mechanisms underlying particular adaptations which in many cases, phylogenetically related microorganisms differ more across environments than across taxa (Lee et al., 2012). *Virgibacillus* genus comprises gram-positive salt tolerant bacterial strains which genes could reveal osmoadaptation mechanisms and genes involved in various unique metabolic pathways. Pangenome was obtained and further functional analysis of exclusive genes was performed. Additionally, we gathered information from the environmental variables corresponding to the sampling sites of each reported organism and constructed an all genomes phylogeny to understand the relationship between functions, environment and its evolutionary history.

**Project goal**

Characterize potential local adaptations of the *Virgibacillus* genus associated with the environment from its functional profile and exclusive genes obtained by its pangenomic analysis.

**Methodology**

![alt text](https://github.com/solouli/Pangenomics_Adaptations/blob/master/methodology.png)

1) Genomic assembly of *Virgibacillus sp* from Cuatro Ciénegas, Coahuila.

2) Phylogeny from multiple alignments using the core genome.

3) Pangenome analysis: core and accessory.

4) Functional annotation of exclusive fraction (singletons) of the pangenome.

**Acknowledgments**

This work is supported by CONACYT-SEP-Project 220708 and DGAPA-PAPIIT, UNAM-Project IG200319. Appreciation for their help and support to Erika Aguirre-Planter and Laura Espinosa-Asuar.

## Pangenomics_Adaptations is a respository for the code used in this project.
